src/Bundle/Resources/config/services/listener.xml

```diff
@@ -15,66 +15,66 @@
+        <service id="sylius.listener.form" class="Sylius\Resource\Symfony\EventListener\FormListener">
             <argument type="service" id="sylius.resource_metadata_operation.initiator.http_operation" />
             <argument type="service" id="sylius.context.initiator.request_context" />
             <argument type="service" id="sylius.form.factory" />
             <tag name="kernel.event_listener" event="kernel.view" priority="128" />
         </service>
 
-        <service id="sylius.listener.validate" class="Sylius\Component\Resource\Symfony\EventListener\ValidateListener">
+        <service id="sylius.listener.validate" class="Sylius\Resource\Symfony\EventListener\ValidateListener">
             <argument type="service" id="sylius.resource_metadata_operation.initiator.http_operation" />
             <argument type="service" id="validator" />
             <tag name="kernel.event_listener" event="kernel.view" priority="64" />
         </service>
 
-        <service id="sylius.listener.write" class="Sylius\Component\Resource\Symfony\EventListener\WriteListener">
+        <service id="sylius.listener.write" class="Sylius\Resource\Symfony\EventListener\WriteListener">
             <argument type="service" id="sylius.resource_metadata_operation.initiator.http_operation" />
             <argument type="service" id="sylius.context.initiator.request_context" />
             <argument type="service" id="sylius.state.processor" />
             <tag name="kernel.event_listener" event="kernel.view" priority="32" />
         </service>
 
-        <service id="sylius.listener.serialize" class="Sylius\Component\Resource\Symfony\EventListener\SerializeListener">
+        <service id="sylius.listener.serialize" class="Sylius\Resource\Symfony\EventListener\SerializeListener">
             <argument type="service" id="sylius.resource_metadata_operation.initiator.http_operation" />
             <argument type="service" id="serializer" on-invalid="null" />
             <tag name="kernel.event_listener" event="kernel.view" priority="16" />
         </service>
 
-        <service id="sylius.listener.flash" class="Sylius\Component\Resource\Symfony\EventListener\FlashListener">
+        <service id="sylius.listener.flash" class="Sylius\Resource\Symfony\EventListener\FlashListener">
             <argument type="service" id="sylius.resource_metadata_operation.initiator.http_operation" />
             <argument type="service" id="sylius.context.initiator.request_context" />
             <argument type="service" id="sylius.helper.flash" />
             <tag name="kernel.event_listener" event="kernel.view" priority="16" />
         </service>
 
-        <service id="sylius.listener.respond" class="Sylius\Component\Resource\Symfony\EventListener\RespondListener">
+        <service id="sylius.listener.respond" class="Sylius\Resource\Symfony\EventListener\RespondListener">
```

Comment:
Checking just in case,
Is `Sylius\Component\Resource\Symfony\Validator\EventListener\ValidationExceptionListener` class below moveable as well?

Reply:
done